src/isonline.js

```diff
@@ -37,6 +41,13 @@ function isOnlineCli (...urls) {
   });
 }
 
+export function getClusterUrls (clusterName) {
+  const nodes = nmo.config.get(clusterName);
+  if (!nodes) {
+    throw new Error('Cluster does not exist');
```

Comment:
can you set the error type to EUSAGE?

using EUSAGE as error type hides the long stacktrace and the message that the user should submit an issue in the bugtracker

example: https://github.com/garrensmith/couchdb-nmo/blob/isonline-cluster/src/isonline.js#L14

code handling error types: https://github.com/garrensmith/couchdb-nmo/blob/isonline-cluster/bin/nmo-cli.js#L39-L59